dll/win32/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ add_subdirectory(ieframe)
 add_subdirectory(iernonce)
 add_subdirectory(imaadp32.acm)
 add_subdirectory(imagehlp)
+#add_subdirectory(imageres)
```

Comment:
@Splitwirez As you can see by opening the link to this PR, you can observe that the imageres is still there. You might have forgotten to `git rm ...` it , and `git commit --amend` it in the PR (+ `git push --force-with-lease`)